qamomile/core/converters/converter.py

```diff
@@ -80,9 +81,11 @@ def __init__(
             compiled_instance: The compiled instance of the optimization problem.
             relax_method (jmt.pubo.RelaxationMethod): The relaxation method for PUBO conversion.
                 Defaults to AugmentedLagrangian.
+            normalize_model (bool): Normalize objective function.
```

Comment:
I think the explanation here is a little inaccurate (I know this is same as `jijmodeling_transpiler`)
I guess the explanation below is more accurate.

> The objective function and the constraints are normalized using the maximum absolute value of the coefficients contained in each.

Reply:
So do I just change it on the local and then push it to the same branch again?